docs/settings/security-settings.asciidoc

```diff
@@ -26,6 +26,112 @@ screens, and authorization using <<kibana-privileges>>. To disable
 Set to `true` to enable audit logging for security events. By default, it is set
 to `false`. For more details see <<xpack-security-audit-logging>>.
 
+[float]
+[[authentication-security-settings]]
+==== Authentication security settings
+
+You configure authentication settings in the `xpack.security.authc` namespace in `kibana.yml`.
+
+For example:
+
+[source,yaml]
+----------------------------------------
+xpack.security.authc:
```

Comment:
__issue:__ unfortunately we didn't support this config format in 7.3, it was only [added in 7.7](https://github.com/elastic/kibana/pull/53010). Before 7.7 `xpack.security.authc.providers` was basically [an array of strings](https://www.elastic.co/guide/en/kibana/7.3/kibana-authentication.html#token-authentication), e.g.:
```yaml
xpack.security.authc.providers: [saml, basic, oidc]
```

The setting specifies the list of authentication providers types and the order here determines the order of the provider in the authentication chain.

In addition to that in 7.3 and 7.4 we only had `xpack.security.authc.saml.realm: saml1` (SAML realm in Elasticsearch that `saml` provider should use if mentioned in `xpack.security.authc.providers`) and `xpack.security.authc.oidc.realm: oidc1` (OpenID Connecy realm in Elasticsearch that `oidc` provider should use if mentioned in `xpack.security.authc.providers`).

In [7.5 we also added `xpack.security.authc.saml.maxRedirectURLSize`](https://github.com/elastic/kibana/pull/44513) that we deprecated [in 7.7](https://github.com/elastic/kibana/pull/53010) (with the introduction of the new config format I mentioned above) in favor of `xpack.security.authc.providers.saml.<provider-name>.maxRedirectURLSize`. The `xpack.security.authc.providers.saml.<provider-name>.maxRedirectURLSize` was eventually deprecated [in 7.10](https://github.com/elastic/kibana/pull/68117) and scheduled for removal in 8.0 (unfortunately I forgot to mention this deprecation in the PR description). This settings was basically a workaround that likely no one used in practice, so maybe we don't really need to document it. Sorry for all the mess with these deprecations!